.github/workflows/pkgci_test_pjrt.yml

```diff
@@ -27,11 +27,10 @@ jobs:
         # TODO: cuda runner is not available yet, refer to #18814
         #   - runner: some-cuda-available-runner
         #     pjrt_platform: cuda
-        # TODO: enable these AMD runners
-        #   - runner: nodai-amdgpu-w7900-x86-64
-        #     pjrt_platform: rocm
-        #   - runner: nodai-amdgpu-w7900-x86-64
-        #     pjrt_platform: vulkan
+          - runner: nodai-amdgpu-w7900-x86-64
+            pjrt_platform: rocm
+          - runner: nodai-amdgpu-w7900-x86-64
+            pjrt_platform: vulkan
```

Comment:
I'd add `fail-fast: false` to this `matrix` so  failures on one runner/platform don't cancel the other jobs early: 
![image](https://github.com/user-attachments/assets/16c39254-4ce7-47f9-be46-f36b6ed191af)

Reply:
Ahh thank you. Added now.